.github/workflows/deploy-storybook.yml

```diff
@@ -1,24 +1,35 @@
 name: 'Deploy Storybook'
 
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+  workflow_call:
+    inputs:
+      version:
+        description: Latest elements version in format v[0-9].[0-9].[0-9]
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Latest elements version in format v[0-9].[0-9].[0-9]
+        required: true
+        type: string
```

Comment:
When manually triggering the workflow, it would be great to just do it without providing a version in case you just want to fix some storybook-related stuff.

I expect the workflow to work roughly like this:
```
- install deps
- build storybook
- update symlink (only if a version was provided)
- update hosted versions (only if a version was provided)
- Deploy storybook (Commit message = version provided ? 'Deploy version xyz' : 'Update Storybook')
```